ci/test-elk-flow-collector.sh

```diff
@@ -4,18 +4,27 @@ LOGSTASH_PORT=4739
 LOGSTASH_IP="0.0.0.0"
 LOGSTASH_PROTOCOL="udp"
 FAILURE=false
+export KUBECONFIG=${WORKSPACE}/jenkins/out/kubeconfig
```

Comment:
can we make sure that the script can be used locally on an arbitrary K8s cluster? maybe look at the other CI scripts (e.g. https://github.com/vmware-tanzu/antrea/blob/master/ci/test-conformance-eks.sh) for inspiration. Ideally one should be able to provide the path to kubeconfig as a command-line argument and the script should not depend on the `WORKSPACE` en variable to be set.

Reply:
Updated. Thanks!